config/crd/bases/datamover.oadp.openshift.io_volumesnapshotbackups.yaml

```diff
@@ -17,7 +17,20 @@ spec:
     singular: volumesnapshotbackup
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - jsonPath: .status.sourcePVCData.name
+      name: PVC Name
+      type: string
+    - jsonPath: .spec.volumeSnapshotContent.name
+      name: VolumeSnapshotContent
+      type: string
+    - jsonPath: .status.phase
+      name: Phase
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
```

Comment:
Why is use `Age` which is different from `creationTimestamp`?

If it is truly a type of date.. it won't be displaying age (number of days/months/years) but rather a date (date/month/year) that VSB was created on.